lib/kino.ex

```diff
@@ -116,20 +116,35 @@ defmodule Kino do
   `inspect/2`.
   '''
 
+  import Kernel, except: [inspect: 1]
+
   @type nothing :: :"do not show this result in output"
 
   @doc """
-  Sends the given term as cell output.
+  Renders the given term as cell output.
 
-  This allows any Livebook cell to have multiple evaluation
-  results. You can think of this function as a generalized
-  `IO.puts/2` that works for any type.
+  This effectively allows any Livebook cell to have multiple
+  evaluation results.
   """
-  @spec render(term()) :: nothing()
+  @spec render(term()) :: term()
   def render(term) do
     output = Kino.Render.to_livebook(term)
     Kino.Bridge.put_output(output)
-    nothing()
+    term
+  end
+
+  @doc """
+  Inspects the given term as cell output.
+
+  This works essentially the same as `IO.inspect/2`, except it
+  always produces colored text and respects the configuration
+  set with `configure/1`.
+  """
+  @spec inscpect(term()) :: term()
+  def inscpect(term) do
```

Comment:
We already have `configure/1` for setting options, so not sure if we want options here as well?

Reply:
We may want options like `structs: false` when debugging certain scenarios.